scripts/actions/remove-completed-batch.js

```diff
@@ -22,11 +23,10 @@ const saveRemainingBatches = async () => {
     console.log(`[!] Unable to delete all contexts`);
   }
 
-  await saveToTranslationQueue(
-    { type: 'being_translated' },
-    'set batchUids = :batchUids',
-    { ':batchUids': batchUids }
-  );
+  for (const batch_uid of completedBatches) {
+    const job = await getJobs({ batch_uid });
+    updateJob(job.id, { status: 'IN_REVIEW' });
```

Comment:
i think this will need an await. also, i believe status needs to be an integer since its a numeric value in the table.

if we want, we can change that. but it might be just as easy to create or refer to an enum somewhere (probably database.js) so that you can do something like:
```js
await updateJob(job.id,  { status: statuses.IN_REVIEW }
```

Reply:
Thanks for the review. I can add an `await`, but is it necessary if nothing else depends on it? Also with `await`, might a long update or one that errors out block the remaining updates in a for loop?

I couldn't find the enum mapping for statuses. I did find in `/scripts/models/status.js` that Status.status looks like is will be a `String` in this scope. Here is a snippet of the Status structure:
 
```javascript
module.exports = (sequelize, DataTypes) => {
  const status = sequelize.define(
    'status',
    {
      id: {
        type: DataTypes.INTEGER,
        allowNull: false,
        primaryKey: true,
        autoIncrement: true,
      },
      status: {
        type: DataTypes.TEXT,
        allowNull: false,
        unique: true,
      },
    },
    {
      tableName: 'statuses',
    }
  );

  return status;
};

Reply:
1. for async, good call. i'll defer to you since my knowledge on js is limited. mainly i just want to make sure we capture errors if any occur. if we need await, and want the concurrency, i think we could alter that to use map instead with promise.all, and await that instead. if not, forget i said anything.

2. this is kind of my bad 🙈  you can ignore what i said here (the documentation i have for the method says numeric, but thats incorrect).

Reply:
Oh good point. We're not really `catch`ing any errors here (or logging successes). Perhaps we should have wrapped this in a `try` / `catch` block and done something like this for the success case:

```js
for (const batch_uid of completedBatches) {
  const job = await getJobs({ batch_uid });
  await updateJob(job.id, { status: 'IN_REVIEW' });
  console.log(`Status of job ${job.id} updated to ${job.status}`);
}
```